basic/src/tasks/TaskDashboardPage.tsx

```diff
@@ -0,0 +1,19 @@
+import { type AuthUser } from "wasp/auth";
+import { CreateTaskForm } from "./components/CreateTaskForm";
+import { TaskList } from "./components/TaskList";
+
+export const TaskDashboardPage = ({ user }: { user: AuthUser }) => {
+  return (
+    <div className="flex flex-col items-center gap-12 p-12 py-24">
+      <h1 className="text-4xl font-bold">{`${user.username}'s tasks 📋`}</h1>
```

Comment:
```suggestion
      <h1 className="text-4xl font-bold">{`${user.username}'s tasks`}</h1>
```
I found not using emojis to look cleaner in the end, so this is just my personal preference.

Reply:
Sad emoji noises.